src/current/v25.4/log-sql-activity-to-datadog.md

```diff
@@ -7,6 +7,12 @@ docs_area: manage
 
 This tutorial describes how to configure logging of telemetry events, including [`sampled_query`]({% link {{ page.version.version }}/eventlog.md %}#sampled_query) and [`sampled_transaction`]({% link {{ page.version.version }}/eventlog.md %}#sampled_query), to [Datadog](https://www.datadoghq.com/) for finer granularity and long-term retention of SQL activity. The `sampled_query` and `sampled_transaction` events contain common SQL event and execution details for [statements]({% link {{ page.version.version }}/sql-statements.md %}) and [transactions]({% link {{ page.version.version }}/transactions.md %}).
 
+{{site.data.alerts.callout_info}}
+`sampled_query` and `sampled_transaction` events currently sent to the [`TELEMETRY`]({% link {{ page.version.version }}/logging-overview.md %}#telemetry) channel will be routed to the [`SQL_EXEC`]({% link {{ page.version.version }}/logging-overview.md %}#sql_exec) channel in a future release.
+
+To assess potential downstream impacts on your logging setup and pipelines, set the [`log.channel_compatibility_mode.enabled`]({% link {{ page.version.version }}/logging-overview.md %}#log-channel_compatibility_mode-enabled) cluster setting to `false` in a non-production environment. This will log these events to the [`SQL_EXEC`]({% link {{ page.version.version }}/logging-overview.md %}#sql_exec) channel.
```

Comment:
Wonder if it's better to direct readers to the heading `#log-channel_compatibility_mode-enabled` in each of these sections instead of suggesting they enable this with more limited context. This is the area where I'd need to take a closer look with more time to review, but let me know if this comment alone is helpful?